src/utils/arappUtils.ts

```diff
@@ -25,28 +25,34 @@ export function readArappIfExists(): AragonAppJson | undefined {
 
 /**
  * Returns app ens name.
+ * @param network "mainnet"
  * @return "voting.open.aragonpm.eth"
  */
-export function getAppEnsName(): string {
+export function getAppEnsName(network = 'default'): string {
   const arapp = readArapp()
 
-  const defaultEnvironment = arapp.environments.default
-  if (!defaultEnvironment) {
-    throw new BuidlerPluginError('Default environemnt not found in arapp.json')
+  const environment: AragonEnvironment | undefined =
+    arapp.environments[network] || arapp.environments['default']
+  if (!environment) {
+    throw new BuidlerPluginError(`Default environemnt not found in arapp.json.`)
   }
 
-  const appName = defaultEnvironment.appName
-  if (!appName) throw Error('No appName found in environment')
+  const appEnsName = (environment || {}).appName
+  if (!appEnsName)
+    throw new BuidlerPluginError(
+      `No appName found. You need to setup one for the environment ${environment} in the arapp.json file.`
```

Comment:
Recommend rewording as:

> No appName found in your arapp.json for the ${environment} environment. Please add one.

And maybe point to docs on the arapp.json file for what value makes sense here.

Reply:
Can you review the updated errors in this PR? https://github.com/aragon/buidler-aragon/pull/120